src/app/views/query-runner/QueryInput.tsx

```diff
@@ -28,7 +30,9 @@ export const QueryInputControl = ({
       <PrimaryButton
         onClick={() => handleOnRunQuery()}
       >
-        Run Query
+        <FormattedMessage
```

Comment:
I really like how a localized string resource is referenced like this

Reply:
Me too.